src/main/java/com/meilisearch/sdk/IndexesHandler.java

```diff
@@ -4,7 +4,11 @@
 import com.meilisearch.sdk.model.Task;
 import java.util.HashMap;
 
-/** Wrapper around the HttpClient class to ease usage for Meilisearch indexes */
+/**
+ * IndexesHandler class covering all the methods related to an index
```

Comment:
Documents, which covers the documents API, is called `documents` but indexes, is cal.ed `IndexesHandler`. Is there a reason?

```suggestion
 * Class covering the Meilisearch Index API.
```